ethdb/database.go

```diff
@@ -95,13 +95,13 @@ type AncientReader interface {
 	AncientSize(kind string) (uint64, error)
 }
 
-// AncientBatchReader is the interface for 'batched' or 'atomic' reading.
-type AncientBatchReader interface {
-	AncientReader
+// AncientReader is the extended ancient reader interface including 'batched' or 'atomic' reading.
+type AncientReader interface {
```

Comment:
Heh, this is somewhat confusing... So, 

- `AncientBatchReader` -> `AncientReader`
- `AncientReader` -> `AncientReaderOp`

But semantically, the interfaces didn't change. Except that you've added some `...Stater` interfaces. LGTM

Reply:
The renaming doesn't change anything, semantically. I just want to align with the writer `AncientWriterOp` and `AncientWriter`.

And I think `AncientReaderOp ` and `AncientReader ` are also acceptable, so just make the renaming.